Support for matrix notifications (#9196)
netdata/netdata: health/notifications/matrix/README.md

> # matrix.org - riot.im

Reviewer: Let's reference these by their formal names, not their URLs.

```suggestion
# Matrix and Riot
```

> Send notifications to matrix network rooms.

Reviewer: I think we should be a bit more explicit about the services used in this method.

```suggestion
Send notifications to [Matrix](https://matrix.org/) network rooms using the [Riot](https://about.riot.im/) client.
```

Author: I haven't found Riot requirement inside the code, so I don't think this change should be applied

Reviewer: Fair enough! I think I didn't fully understand the relationship between the two when I first started reviewing. Let's still please capitalize **Matrix**, but you can indeed disregard the other requests.

Author: Thanks!. Done.

> The requirements for this connector are

Reviewer: ```suggestion
The requirements for this notification method are:
```

> 1. The url of the homeserver (https://homeserver:port)

Reviewer 2: @joelhans markdown linter doesnt like this URL

Author: I know, but it's not real URL :) it's example

Reviewer: It should be in an inline code block to prevent the link-checker from trying to follow it.

```suggestion
1. The url of the homeserver (`https://homeserver:port`)
```

> 2. Credentials for connecting to the homeserver, in the form of a valid accesstoken for your account (or for a dedicated notification account). These tokens usually don't expire.

Reviewer 3: ```suggestion
2. Credentials for connecting to the homeserver, in the form of a valid access token for your account (or for a dedicated notification account). These tokens usually don't expire.
```

3. The room ids that you want to sent the notification to.

To obtain the access token you can use the following curl command:
```
curl -XPOST -d '{"type":"m.login.password", "user":"example", "password":"wordpass"}' "https://homeserver:8448/_matrix/client/r0/login"
```

> The room ids are unique identifiers and can be obtained from the room settings in a matrix client (riot). Their format is !uniqueid:homeserver.

Reviewer: ```suggestion
The room ids are unique identifiers and can be obtained from the room settings in a Matrix client (Riot). Their format is !uniqueid:homeserver.
```

Multiple room ids can be defined by separating with a space character.

> Detailed information about the matrix client api is available at the official [site](https://matrix.org/docs/guides/client-server.html)

Reviewer: ```suggestion
Detailed information about the Matrix client API is available at the official [site](https://matrix.org/docs/guides/client-server.html)
```
